### PR TITLE
Make radio URL configurable in Thread tests

### PR DESCRIPTION
**Pull request by jpm-canonical**
## Summary

Solves #62

* Add configuration for the local and remote radio URLs
* Refactor constants out to a dedicated file

## Testing Steps

* Check if test is passing in github actions

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -24,9 +32,14 @@ REMOTE_HOST="192.168.178.95" \
...
-
-Some environment variables can modify the test functionality. Refer to these in
-[the documentation](https://pkg.go.dev/github.com/canonical/matter-snap-testing/env)
-of the `matter-snap-testing` Go package.
-
+### Thread specific environment variables
+
+ Variable name    | Required | Default value                   | Description                       
+------------------|----------|---------------------------------|-----------------------------------
+ LOCAL_INFRA_IF   | no       | wlan0                           | wlan0                             | Local backhaul network interface
```
Comment (farshidtz) [inline tests/README.md:39]:
```suggestion
 LOCAL_INFRA_IF   | no       | wlan0                           | Local backhaul network interface  
```

File: tests/README.md [hunk 2]
```diff
@@ -24,9 +32,14 @@ REMOTE_HOST="192.168.178.95" \
...
-Some environment variables can modify the test functionality. Refer to these in
-[the documentation](https://pkg.go.dev/github.com/canonical/matter-snap-testing/env)
-of the `matter-snap-testing` Go package.
-
+### Thread specific environment variables
+
+ Variable name    | Required | Default value                   | Description                       
+------------------|----------|---------------------------------|-----------------------------------
+ LOCAL_INFRA_IF   | no       | wlan0                           | wlan0                             | Local backhaul network interface  
+ LOCAL_RADIO_URL  | no       | spinel+hdlc+uart:///dev/ttyACM0 | Local RCP serial port
```
Comment (farshidtz) [inline tests/README.md:40]:
```suggestion
 LOCAL_RADIO_URL  | no       | spinel+hdlc+uart:///dev/ttyACM0 | Local RCP URL             
```

File: tests/README.md [hunk 3]
```diff
@@ -24,9 +32,14 @@ REMOTE_HOST="192.168.178.95" \
...
+
+ Variable name    | Required | Default value                   | Description                       
+------------------|----------|---------------------------------|-----------------------------------
+ LOCAL_INFRA_IF   | no       | wlan0                           | wlan0                             | Local backhaul network interface  
+ LOCAL_RADIO_URL  | no       | spinel+hdlc+uart:///dev/ttyACM0 | Local RCP serial port             
+ REMOTE_HOST      | yes      |                                 | Remote device IP or hostname      
+ REMOTE_USER      | yes      |                                 | Remote device SSH username        
+ REMOTE_PASSWORD  | yes      |                                 | Remote device SSH password        
+ REMOTE_INFRA_IF  | no       | wlan0                           | Remote backhaul network interface 
+ REMOTE_RADIO_URL | no       | spinel+hdlc+uart:///dev/ttyACM0 | Remote RCP serial port
```
Comment (farshidtz) [inline tests/README.md:45]:
```suggestion
 REMOTE_RADIO_URL | no       | spinel+hdlc+uart:///dev/ttyACM0 | Remote RCP URL            
```

**Review by farshidtz — CHANGES_REQUESTED** [review]
A few minor documentation issues.
Can't run the tests right now because snap store return 503.

**Comment by jpm-canonical** [comment]
This PR was used to test #67. Both generic and thread tests passed, using `/dev/ttyACM1` on local host.

**Review by farshidtz — APPROVED** [review]
I also just tested and it works as expected.

**Comment by farshidtz** [comment]
Updated the PR title.

**Pull request merged**